pyscriptjs/tests/integration/test_py_terminal.py

```diff
@@ -131,3 +131,18 @@ def test_config_false(self):
         )
         term = self.page.locator("py-terminal")
         assert term.count() == 0
+
+    def test_config_docked(self):
+        """
+        config.docked == "docked" is also the default: a <py-terminal auto docked> is
+        automatically added to the page
+        """
+        self.pyscript_run(
+            """
+            <button id="my-button" py-onClick="print('hello world')">Click me</button>
+            """
+        )
+        term = self.page.locator("py-terminal")
+        self.page.locator("button").click()
+        expect(term).to_be_visible()
+        assert term.get_attribute("docked") == ""
```

Comment:
this is good as a minimal test, but if I understand correctly it doesn't test the "docking" itself.
I don't really know what is the proper way to test these properties, but ideally I would like a tests to check:
1. that when it's `docked`, it's actually and "visually" at the bottom of the page
2. than when I manually add a non-docked terminal, it's not docked
3. that when I have a docked terminal and a long page, I can scroll to the bottom of the page and the terminal doesn't hide it

Reply:
> I don't really know what is the proper way to test these properties

you are kinda asking to test how CSS works :-) ... are we testing every `padding` or `margin` we have in CSS?

A better question to me around this PR is: what version of browsers are we exactly targeting? We need to be sure these supports `:has(...)` selector, as example, because while playwright might have a chrome version that does, some user might not have that same version or higher, meaning that a visual test with a specific browser and a specific browser version will not guarantee anything concretely for our users.

> than when I manually add a non-docked terminal, it's not docked

this is CSS ... the JS does not dock anything. If the attribute is not there, the CSS doesn't dock it.

> that when I have a docked terminal and a long page, I can scroll to the bottom of the page and the terminal doesn't hide it

This is also still CSS. The padding bottom to the top most component + the fixed position gives you that out of the box.

Reply:
P.S. it looks like we have various options to screenshot here https://playwright.dev/python/docs/screenshots so, if this makes sense but worth mentioning this test would still not guarantee the CSS works in all target browsers (a detail I should've probably asked already).

Reply:
OK, if I understood correctly, we're going to think about a proper solution for both target browsers and screenshots later on with the rest of the team. Resolving this for now, if that's OK.